File: app/src/main/java/org/lemonadestand/btb/features/login/models/TangoAccount.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

public class TangoAccount {
    @SerializedName("accountIdentifier")
    String accountIdentifier;

    @SerializedName("accountNumber")
    String accountNumber;

    @SerializedName("displayName")
    String displayName;

    @SerializedName("currencyCode")
    String currencyCode;

    @SerializedName("currentBalance")
    int currentBalance;

    @SerializedName("createdAt")
    String createdAt;

    @SerializedName("status")
    String status;

    @SerializedName("contactEmail")
    String contactEmail;


    public void setAccountIdentifier(String accountIdentifier) {
        this.accountIdentifier = accountIdentifier;
    }
    public String getAccountIdentifier() {
        return accountIdentifier;
    }

    public void setAccountNumber(String accountNumber) {
        this.accountNumber = accountNumber;
    }
    public String getAccountNumber() {
        return accountNumber;
    }

    public void setDisplayName(String displayName) {
        this.displayName = displayName;
    }
    public String getDisplayName() {
        return displayName;
    }

    public void setCurrencyCode(String currencyCode) {
        this.currencyCode = currencyCode;
    }
    public String getCurrencyCode() {
        return currencyCode;
    }

    public void setCurrentBalance(int currentBalance) {
        this.currentBalance = currentBalance;
    }
    public int getCurrentBalance() {
        return currentBalance;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }
    public String getCreatedAt() {
        return createdAt;
    }

    public void setStatus(String status) {
        this.status = status;
    }
    public String getStatus() {
        return status;
    }

    public void setContactEmail(String contactEmail) {
        this.contactEmail = contactEmail;
    }
    public String getContactEmail() {
        return contactEmail;
    }


}

File: app/src/main/java/org/lemonadestand/btb/models/TangoAccount.kt
package org.lemonadestand.btb.models

import com.google.gson.annotations.SerializedName

class TangoAccount {
	@SerializedName("accountIdentifier")
	var accountIdentifier: String? = null

	@SerializedName("accountNumber")
	var accountNumber: String? = null

	@SerializedName("displayName")
	var displayName: String? = null

	@SerializedName("currencyCode")
	var currencyCode: String? = null

	@SerializedName("currentBalance")
	var currentBalance: Int = 0

	@SerializedName("createdAt")
	var createdAt: String? = null

	@SerializedName("status")
	var status: String? = null

	@SerializedName("contactEmail")
	var contactEmail: String? = null
}
